CHANGELOG

```diff
@@ -16,6 +16,7 @@
  * Bump minimum PHP version to 8.0
  * Fix integration tests when a test has more than one data/expect section and deprecations
  * Add the `enum_cases` function
+ * Add the `types` tag
```

Comment:
```suggestion
 * Add the `types` tag (experimental)
```

Reply:
I've just bumped the version to 3.13 so that we can merge this PR. Can you rebase and move this line to the 3.13 section?